projects/igniteui-angular/src/lib/core/styles/components/date-range-picker/_date-range-picker-theme.scss

```diff
@@ -65,19 +59,6 @@
         align-items: center;
         color: var-get($theme, 'label-color');
         margin: 0 rem(8px);
-        padding-top: rem(18px, map.get($base-scale-size, 'comfortable'));
-    }
-
-    %igx-date-range-picker--cosy {
```

Comment:
You can also remove the `%igx-date-range-picker--cosy` and `%igx-date-range-picker--compact` selectors from the date range picker component.